helper/pom.xml

```diff
@@ -89,6 +89,7 @@
                                     <include>org.spongepowered:configurate-hocon</include>
                                     <include>com.typesafe:config</include>
                                     <include>net.kyori:event-api</include>
+                                    <inclue>org.slf4j:*</inclue>
```

Comment:
> `inclue`

Reply:
the latest version of Jedis doesn't package slf4j

Reply:
Should be `include` not `inclue`.